ANEVIS-375 #comment Fix typo (#2)
anevis-solutions/github-test: README.md

> # github-test

Author: I like the title for 2 reasons

1. it's short
2. it has three `t`s

I have more information attached below
[why-this-title-is-great.txt](https://github.com/anevis-solutions/github-test/files/6703361/why-this-title-is-great.txt)

Author: jira test [ANEVIS-375]

Author: Ne, das ist Quatsch, passt so.

Author: a1a2343af463c63ce0854109c7de148285a9a431

Author: [ANEVIS-375]

> This is testing Repo for Github.

Author: This reminds me of #1 and [ANEVIS-1] a little, are we sure about changing this?

> This is a testing Repo for Github.

Author: Are we sure `a` is the best choice here?

Author: I agree, what do you think about `the`?
```suggestion
This is the testing Repo for Github.
```

Reviewer: I think `the` makes sense here since it is the only one and no other testing repo will be added (probably).

> This is a testing Repo for Github.

Author: Das ist ein Kommentar

> This is a testing Repo for Github.
> We should add more information here.

Author: ```suggestion
This is the testing Repo for Github.
We shouldn't add more information here.
```

> Maybe even more.

Reviewer: ```suggestion
Maybe even way more.
```

> This is the testing Repo for Github.

Reviewer: My review comment

Author: @ralf-lackinger ?

> This is the testing Repo for Github.

Author: Kommentar

> Maybe even way more.

Author: Diese Zeilen sind nicht gut formatiert

> We shouldn't add more information here.

Author: Kommentare 2

> Maybe even way more.

Reviewer: My comment

> We shouldn't add more information here.
> Maybe even way more.

Author: ```suggestion
We should not add more information here.
Maybe even way more.
```
